app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -284,13 +288,31 @@ def host_subscription_facet(host)
         host.subscription_facet
       end
 
+      def skip_task?(task)
+        # Skip task that doesn't apply errata
+        input = get_task_input(task)
+        input.blank? || input['host'].blank?
+      end
+
+      def get_task_input(task)
+        @_tasks_input[task.id] ||= if task.label == 'Actions::Katello::Host::Erratum::ApplicableErrataInstall'
+          task.execution_plan_action.all_planned_actions(Actions::Katello::Host::Erratum::Install).first.try(:input) || {}
+        else
+          task.input
+        end
+      end
+
       def parse_errata(task)
-        @_tasks_errata_cache[task.id] ||= if task.input['errata'].present?
+        @_tasks_errata_cache[task.id] ||= if get_task_input(task)['errata'].present?
                                             # katello agent errata
-                                            task.input['errata']
+                                            get_task_input(task)['errata']
                                           else
```

Comment:
As part of the katello-ification of katello-agent actions, this task input key has changed to 'content'.    However i think we should support old and new.  Something like this should work:

```suggestion
        task_input = get_task_input(task)
        agent_input = task_input['errata'] || task_input['content']
        @_tasks_errata_cache[task.id] ||= if agent_input.present?
                                            # katello agent errata
                                            agent_input
                                          else
```

Reply:
Done and also fixed the rubocop errors.

Reply:
looks like there's still a couple of rubocop errors:

[2021-05-09T06:12:30.591Z] /home/jenkins/workspace/katello-pr-test/app/lib/katello/concerns/base_template_scope_extensions.rb:212:120: C: Style/LineEndConcatenation: Use \ instead of + or << to concatenate those strings.

[2021-05-09T06:12:30.591Z]           condition = ["state = 'stopped' AND ((label = 'Actions::RemoteExecution::RunHostJob' AND templates.id = ?) " +

[2021-05-09T06:12:30.591Z]                                                                                                                        ^

[2021-05-09T06:12:30.591Z] /home/jenkins/workspace/katello-pr-test/app/lib/katello/concerns/base_template_scope_extensions.rb:261:39: C: Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.

[2021-05-09T06:12:30.591Z]                 :status => task.result,